elastic/kibana
# [Alerting] Enable rule import/export and allow rule types to exclude themselves from export

Resolves https://github.com/elastic/kibana/issues/101270

## Summary

This PR removes the feature flag for enabling import/export of rules and takes advantage of the [new `isExportable` SO API](https://github.com/elastic/kibana/pull/101860) to exclude security rule types and stack monitoring rule types from being exported.

It adds an `isExportable` flag to the `AlertType` type so that rule types can indicate whether or not they want they want to be exported from the SO Management UI at registration time. While the rule will still show up in the SO Management UI, a message will be logged for non-exportable rule types and the export file will include a listing and count of rules that were excluded from the export.

## To Verify
1. Create some test rules, including stack monitoring & security rules.
2. Initiate a rule export from the SO Management UI
3. Verify that stack monitoring and security rules are excluded from the export file but other rule types are included.


### Checklist

Delete any items that are not applicable to this PR.
